src/core_landice/Registry.xml

```diff
@@ -659,6 +707,14 @@
 			<var name="waterPressure" packages="hydro"/>
                         <var name="channelArea" packages="hydro"/>  <!-- this is only needed if running with channels - could be package-controlled -->
 			<var name="waterFluxMask" packages="hydro"/>
+			<var name="damage"/>
+			<var name="ddamagedt"/>
```

Comment:
Like above, I do not think we need any of these fields to be restart fields other than `damage`.    However, `damageMax` I think would be needed as a restart field and needs to be added here.

Reply:
Those variables are removed and 'damageMax' is added